docs/reference/server-configuration.md

```diff
@@ -33,6 +33,12 @@ For more information on setting properties, see
     environment variable is `KSQL_KSQL_SERVICE_ID`. For more information,
     see [Install ksqlDB with Docker](/operate-and-deploy/installation/install-ksqldb-with-docker).
 
+!!! info
+    The underlying producer and consumer clients in ksqlDB's server can be
```

Comment:
I've been using "ksqlDB Server", but we don't have any hard-and-fast guidance on naming.